en/docs/install.html

```diff
@@ -15,6 +15,7 @@
 {% capture windows      %}{% include_relative _installations/windows.md      %}{% endcapture %}
 {% capture linux        %}{% include_relative _installations/linux.md        %}{% endcapture %}
 {% capture alternatives %}{% include_relative _installations/alternatives.md %}{% endcapture %}
+{% capture nightly      %}{% include_relative _installations/nightly.md      %}{% endcapture %}
```

Comment:
Do you think we should emphasize nightly builds so much? I feel like we should make it a separate thing